SolWatch/Game1.cs

```diff
@@ -107,7 +107,9 @@ protected override void Draw(GameTime gameTime)
                 origin: new Vector2(orbitTexture.Width / 2, orbitTexture.Height / 2),
                 effects: SpriteEffects.None,
                 layerDepth: 0f);
+
```

Comment:
I'd leave this  white space out. spriteBatch.End(); ends the spriteBatch draw thought in my mind.

Reply:
Discussed live; whitespace on line 110 is symmetrical with whitespace on line 84, so we'll keep it in.